src/components/layouts/files/FilesLayout.tsx

```diff
@@ -10,8 +10,8 @@ import MainArea from '../MainArea';
 const FilesLayout = ({ children }: PropsWithChildren) => {
   return (
     <MainArea>
-      <LocationBreadcrumb />
-      <Card>
+      <Card className='w-full h-full sm:px-4 lg:px-6 lg:py-4 p-2 !rounded-none'>
```

Comment:
I guess in this case we could get rid of `<Card>` concept then, as if the card become full screen?

Reply:
updated, do you want me to remove the <card> concept in labs layout as well?

Reply:
That would be great! Thank you!

Reply:
updated